src/plane.js

```diff
@@ -0,0 +1,48 @@
+class Plane {
+
+    constructor(planeID, inFlight) {
+        this.planeID = planeID;
+        this.inFlight = inFlight;
+        this.atAirport = false
+    }
+
+    getPlaneID() {
+        return this.planeID;
+    }
+
+    getInFlight() {
+        return this.inFlight;
+    }
+
+    setInFlight(status) {
+        this.inFlight = status;
+        (status) ? this.setAtAirport(false) : this.setAtAirport(true);
+    }
+
+    getAtAirport() {
+        return this.atAirport;
+    }
+
+    setAtAirport(atAirport) {
+        this.atAirport = atAirport;
+    }
+
+    static checkInputIsAPlaneObject(input) {
```

Comment:
Think this was a good use of static. I could use the method to see if an input was an instance of Plane and since the method was within the Plane class, it keeps it decoupled and encapsulated